server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,13 @@
+const {
+  cohort: { getSpecificCohort },
+} = require('../../../../database/queries');
+
+exports.getSpecificCohort = async (req, res, next) => {
+  try {
+    const { rows } = await getSpecificCohort(req.params.cohortid);
```

Comment:
Use destructuring here
```suggestion
    const {cohortid} = req.params
    const { rows } = await getSpecificCohort(cohortid);
```